cmd/shim/main.go

```diff
@@ -250,6 +250,9 @@ func shim() int {
 		}
 
 		code, err := os.ReadFile(exitCodePath)
+		if os.IsNotExist(err) {
+			return -1
+		}
```

Comment:
Thanks for fixing this along the way, I've seen it pop up here and there but it was always some tertiary failure so I never doubled back.

Reply:
Oh yeah no problem, I had accidentally mounted something at `/` and was very confused for a bit... 😄